.travis.yml

```diff
@@ -40,10 +40,7 @@ before_script:
 
 script:
   - cargo build --verbose --features llvm_stable
-  - make test
-  - git add -A
-  - git diff @
-  - git diff-index --quiet HEAD
+  - cargo test
```

Comment:
you'll need to run `cargo test --features llvm_stable`, otherwise you're compiling with llvm 3.9 support, which can't be done for now (see #70).